src/hooks/useKeyboardNavigation.ts

```diff
@@ -0,0 +1,54 @@
+
+import { useState } from "react";
+import useEventListener from "./useEventListener";
+
+const KEY_HANDLERS = {
+    UP: "onUp",
+    DOWN: "onDown",
+    LEFT: "onLeft",
+    RIGHT: "onRight",
+    ENTER: "onEnter",
+    ESCAPE: "onEscape",
+};
+
+const KEY_CODES = {
```

Comment:
Could KEY_HANDLERS and KEY_CODES be combined?
`
const KEY_HANDLERS = {
    38: "onUp",
};
`

and then

`
const handler = heyBoardEvents[KEY_HANDLERS[keyCode]];
`